src/libraries/System.Formats.Tar/src/System/Formats/Tar/TarEntry.cs

```diff
@@ -56,7 +56,7 @@ internal TarEntry(TarEntry other, TarEntryFormat format)
 
```

Comment:
on line 54, it should also throw `ArgumentException`. This may be a good time to query all the exceptions currently thrown and validate that they are what we want to throw.

Reply:
Done.